canvas_modules/harness/src/client/components/ctrl/CustomEffectsCtrl.jsx

```diff
@@ -76,6 +76,8 @@ class CustomEffectsCtrl extends React.Component {
 	}
 
 	componentDidMount() {
+		this.controller.setAddRemoveRows(this.SOURCE_LIST_ID, false);
```

Comment:
Why do we need to add this?

Reply:
This is added to remove the buttons from the table, to align our examples.
The `add_remove_rows` uihint did not take affect because this control is being created directly through controlFactory. We call `this.setInitialAddRemoveRows()` when setting the form. This is one way to ensure the buttons are hidden for the controls created